src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    private String name;
+    private boolean isWork;
+

```

Comment:
create a consturctor for those two fields, call super() instead of setters in subclasses